packages/relay/package.json

```diff
@@ -7,16 +7,18 @@
     "@kiwicom/fetch": "^2.2.1",
     "@kiwicom/mobile-localization": "^0",
     "@kiwicom/mobile-shared": "^0",
-    "@kiwicom/relay": "^0.27.0"
+    "@kiwicom/relay": "^0.27.0",
+    "@sentry/browser": "^4.6.4"
```

Comment:
browser?

Reply:
Okay, related to this https://github.com/kiwicom/mobile/pull/1509/files#diff-e0465f4baf58d3c87df7993cb549f65fR12

Reply:
Yes, the browser package works. The react-native-package requires setup on the rootView (even outstide of our framework) and we don't want to catch native errors. Just grapqhl errors

Reply:
Also not sure if it is better to use `@sentry/node` but it should be possible with `rn-nodeify`